### PR TITLE
Add FCC models

### PR DESCRIPTION
**Pull request by terasic**
### Company Info

Terasic https://www.terasic.com.tw/

### IoT Plug and Play Device Info

FCC (FPGA Cloud Connectivity Kit) http://fcc.terasic.com.tw/

### Model Submission Goals

Device certification

### CODE REVIEW

File: dtmi/terasic/fcc/de10_nano_threshold-1.json [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1",
+      "@type": "Object",
+      "fields": [
+        {
+          "displayName": {
+            "en": "Minimum Value"
+          },
+          "description" : {
+            "en" : "Minimum sensor value in float to ?????"
```
Comment (digimaun) [inline dtmi/terasic/fcc/de10_nano_threshold-1.json:172]: Just want to confirm that the `?????` value for a few of the description.en properties  in the model are intended?

File: dtmi/terasic/fcc/de10_nano_threshold-1.json [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+          "description" : {
+            "en" : "Maximum sensor value in float to ?????"
+          },
+          "name": "max",
+          "schema": "float"
+        }
+      ]
+    },
+    {
+      "@id": "dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:int;1",
```
Comment (digimaun) [inline dtmi/terasic/fcc/de10_nano_threshold-1.json:190]: Is this interface schema needed? Doesn't look like its currently being used.

Comment (daisukeiot) [reply dtmi/terasic/fcc/de10_nano_threshold-1.json:190]: They are used by tmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:Channel0;1 & tmi:Terasic:FCC:DE10_Nano_Threshold:MPU9250:Channel1;1

Comment (daisukeiot) [reply dtmi/terasic/fcc/de10_nano_threshold-1.json:190]: Channel0/Channel1 are removed from the model.  Checking with Terasic/Intel team.

**Review by digimaun — APPROVED** [review]

**Comment by terasic** [comment]
For the record, two commit were submitted based on the questions from the reviewer with the following changes. 

1. modify the description of dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:float;1

2. delete unused dtmi:Terasic:FCC:DE10_Nano_Threshold:MinMax:int;1

**Pull request merged**